File: 01. JavaScript, NodeJS, ExpressJS, MongoDB, Mongoose/04. JavaScript And ExpressJS - Calculator/models/Calculator.js
function Calculator(leftOperand, operator, rightOperand) {
    this.leftOperand = leftOperand;
    this.operator = operator;
    this.rightOperand = rightOperand;

    this.calculateResult = function () {
        let result = 0;

        switch (operator){
            case '+':
                result = Number(this.leftOperand) + Number(this.rightOperand);
                break;
            case '-':
                result = Number(this.leftOperand) - Number(this.rightOperand);
                break;
            case '*':
                result = Number(this.leftOperand) * Number(this.rightOperand);
                break;
            case '/':
                result = Number(this.leftOperand) / Number(this.rightOperand);
                break;
            case '^':
                result = Math.pow(Number(this.leftOperand), Number(this.rightOperand));
                break;
            case '%':
                result = Number(this.leftOperand) % Number(this.rightOperand);
        }
        return result;
    }
}

module.exports = Calculator;

File: 04. CSharp, ASP.NET/01. CSharp and ASP.NET - Exercises/01. Calculator/Calculator-CSharp/Models/Calculator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Calculator_CSharp.Models
{
    public class Calculator
    {
        public decimal LeftOperand { get; set; }
        public decimal RightOperand { get; set; }
        public char Operator { get; set; }
        public decimal Result { get; set; }
        public Calculator()
        {
            this.Result = 0;
        }
        public Calculator(decimal leftOperand, decimal rightOperand, char calcOperator, decimal result)
        {
            this.LeftOperand = leftOperand;
            this.RightOperand = rightOperand;
            this.Operator = calcOperator;
            this.Result = result;
        }
    }
}